src/web/build.gradle

```diff
@@ -20,6 +20,10 @@ dependencies {
 
     // SL4J
     implementation "org.slf4j:slf4j-simple:1.7.21"
+
+    // Firebase
+    implementation 'com.google.firebase:firebase-admin:7.0.1'
+    implementation platform('com.google.firebase:firebase-bom:26.0.0')
```

Comment:
I haven't looked into it, but do we need the Firebase BoM in there ? Or could it be Android-specific ?

Reply:
You're right, that was something left from my testing